FirebaseStorageSwift/Sources/Storage.swift

```diff
@@ -0,0 +1,206 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+import FirebaseStorageObjC
+import FirebaseCore
+import FirebaseCoreInternal
+import FirebaseAppCheckInterop
+import FirebaseAuthInterop
+
+/**
+ * FirebaseStorage is a service that supports uploading and downloading binary objects,
+ * such as images, videos, and other files to Google Cloud Storage.
+ *
+ * If you call `FirebaseStorage.storage()`, the instance will initialize with the default FirebaseApp,
+ * `FirebaseApp.app()`, and the storage location will come from the provided
+ * GoogleService-Info.plist.
+ *
+ * If you provide a custom instance of FirebaseApp,
+ * the storage location will be specified via the FirebaseOptions#storageBucket property.
+ */
+@objc(FIRStorage) open class Storage: NSObject {
+  // MARK: - Public APIs
+
+  /**
+   * Creates an instance of FirebaseStorage, configured with the custom FirebaseApp @a app.
```

Comment:
What does that `@a` symbol doing there? I have seen it throughout the documentation in this PR.